backend/compact-connect/lambdas/python/staff-users/handlers/users.py

```diff
@@ -188,3 +212,13 @@ def reinvite_user(event: dict, context: LambdaContext):  # noqa: ARG001 unused-a
 
     config.user_client.reinvite_user(email=user['attributes']['email'])
     return {'message': 'User reinvited'}
+
+
+def _verify_caller_is_active(event):
+    caller_sub = event['requestContext']['authorizer']['claims']['sub']
+    # Ensure the caller has not been previously disabled and is calling with an unexpired token
```

Comment:
Let's make this comment a bit more explicit - it's not really about the user being 'disabled' but the db record being removed.